src/index.html

```diff
@@ -7,6 +7,6 @@
   </head>
   <body>
     <div id="root"></div>
-    <script type="module" src="index.jsx"></script>
+    <script type="module" src="./index.jsx"></script>
```

Comment:
One reason why the app won't start from your side when you run `parcel ./src/index.html` is because you have the parcel package installed globally(using `npm i -g`) and locally at the same time(present in package.json).
- Sol: Try to keep only one. You need to uninstall one of them.

Reply:
I have deleted the global version and now run the app locally using the npx parcel src/index.html command.